src/app/validators/currency-max.ts

```diff
@@ -12,7 +12,7 @@ export function getCurrencyMaxValidator(limitOverride?: number): ValidatorFn {
 
       let effectiveLimit = environment.maximumDonationAmount;
```

Comment:
Possibly we should have unit tests for the validators - there aren't any at present, I could start writing some.